ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -354,7 +354,8 @@ def install_master(host, setup_dns=True, setup_kra=False, setup_adtrust=False,
         args.extend([
             '--setup-dns',
             '--forwarder', host.config.dns_forwarder,
-            '--auto-reverse'
+            '--auto-reverse',
+            '--no-dnssec-validation',
```

Comment:
With this change, **all** the integration tests with DNS will be setup with `--no-dnssec-validation`. This breaks `ipatests.test_integration.test_upgrade.TestUpgrade.test_current_named_conf` for instance (see the azure test output).

If the intent is to change only the healthcheck tests, you can modify the `install` method of the class `TestIpaHealthCheck`, so that it calls `tasks.install_master` with `extra_args=['--no-dnssec-validation']`. This way only this test will be impacted.

Reply:
@flo-renaud i have reverted the changes done in tasks.py, will be making the changes in healthcheck tests